packages/core/src/html/NodeProcessor.js

```diff
@@ -123,12 +177,17 @@ class NodeProcessor {
    * @param node Element to transform
    */
   static _transformSlottedComponents(node) {
+    /*
+      This function is called after deprecated vue slot syntax has been converted to shorthand.
+      We only have to handle vue shorthand slot syntax.
+    */
     node.children.forEach((child) => {
-      const slot = child.attribs && child.attribs.slot;
-      if (slot) {
-        // Turns <div slot="content">... into <div data-mb-slot-name=content>...
-        child.attribs['data-mb-slot-name'] = slot;
-        delete child.attribs.slot;
+      // Turns <div #content>... into <div data-mb-slot-name=content>...
+      const vslotShorthand = NodeProcessor.getVslotShorthand(child);
+      if (vslotShorthand) {
+        const slotName = vslotShorthand.substring(1, vslotShorthand.length); // remove # from vslot shorthand
```

Comment:
this seems repeated quite often. how about moving it into `getVslotShorthand`?

Reply:
Hmm yup that's possible!